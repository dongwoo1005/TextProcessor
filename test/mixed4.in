stdin doublewords doublewords doublewords count o
Hello World
