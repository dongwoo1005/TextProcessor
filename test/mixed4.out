Hell1
Hell2
Hell3
Hell4
Hell5
Hell6
Hell7
Hell8
W9rld
W10rld
W11rld
W12rld
W13rld
W14rld
W15rld
W16rld
